javascript/javascript2/week1/countDanishLettersredo.js

```diff
@@ -0,0 +1,19 @@
+const danishString = "Jeg har en blå bil";
+getCountOfDanishLetters(danishString); // returns {total: 1, å: 1}
+
+const danishString2 = "Blå grød med røde bær";
+getCountOfDanishLetters(danishString2); // returns {total: 4, æ: 1, ø: 2, å: 1}
+
+function getCountOfDanishLetters (string) {
+    const danishLettersArray = string.match(/[æåøÆÅØ]/g);
+    let returnObj = {
+        total: danishLettersArray.length,
+    }
+
+    danishLettersArray.forEach(letter => {
+        (returnObj[letter] === undefined) ? returnObj[letter] = 1 : returnObj[letter] += 1;
+    });
+
+    console.log(returnObj)
```

Comment:
Consider this for readabilty:

```suggestion
  const danishLetters = string.match(/[æåøÆÅØ]/g);
  
  const reducer = (acc, letter) => {
    const count = acc[letter] || 0;
    return {
      ...acc,
      [letter]: count + 1,
    };
  };

  return danishLetters.reduce(reducer, {
        total: danishLetters.length,
  });
```